src/lib/db/seed.ts

```diff
@@ -133,6 +140,37 @@ const entitiesContents: Record<string, string[]> = {
 	],
 };
 
+const generateTodo = async (db: Database, persona_id: number, space_id: number) => {
+	const list = await db.repos.entity.create(
+		persona_id,
+		{
+			type: 'Collection',
+			name: 'Grocery List',
+		},
+		space_id,
+	);
+	if (!list.ok) {
+		log.warn('Issue generating todo list');
+		return;
+	}
+	const items = ['eggs', 'milk', 'bread'];
+	for (const item of items) {
+		const result = await db.repos.entity.create(
+			persona_id,
+			{
+				type: 'Note',
+				content: item,
+			},
+			space_id,
+		);
+		if (!result.ok) {
+			log.warn('Issue generating todo item');
+			return;
+		}
+		await db.repos.tie.create(list.value.entity_id, result.value.entity_id, 'HasItem');
```

Comment:
```suggestion
		unwrap(await db.repos.tie.create(list.value.entity_id, result.value.entity_id, 'HasItem'));
```

reason given here: https://github.com/feltcoop/felt-server/pull/300/files#r840193927